### PR TITLE
Add a Python script to convert 2D clawpack data to VTK format 

### PR DESCRIPTION
**Pull request by xinshengqin**
Add a Python script to convert 2D clawpack data to VTK format, which can be visualized with Paraview.

### CODE REVIEW

**Comment by mandli** [comment]
I know that the movie file is small but I am not sure if we want to start adding these to the repository.  I think it might make more sense in the eventual documentation that might be written for this feature though or alternatively provide a URL where the movie is kept.

**Comment by xinshengqin** [comment]
I have removed the *.avi file.

**Comment by rjleveque** [comment]
If you have the sample .avi file in another repository you might want to include a link to it in the README.md file.

File: dev/paraview/claw_to_VTK_2D/README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Convert Clawpack output data to VTK format
 
-The water\_depth.avi in this directory shows an example of visualizing result of
+An example, water\_depth.avi can be found at:
+https://github.com/xinshengqin/claw\_to\_vtk
```
Comment (rjleveque) [inline dev/paraview/claw_to_VTK_2D/README.md]: This link doesn't seem to work as written, remove the back slashes?

Comment (xinshengqin) [inline dev/paraview/claw_to_VTK_2D/README.md]: I am still testing it.

**Comment by xinshengqin** [comment]
I think the link works now.

**Pull request merged**